src/modules/displayData.js

```diff
@@ -0,0 +1,12 @@
+// display the data
+const displayData = (scores) => {
+  const container = document.querySelector('.board');
+  let list = '';
+  scores.result.forEach((item) => {
+    list += `
+        <p>${item.user} : ${item.score} </p>
+      `;
+  });
+  container.innerHTML = list;
+};
+export default displayData;
```

Comment:
- [ ] The data is not displayed the moment the user is inside the app, it has to hit refresh first to see other scores. I think the scores should show from the very beginning